### PR TITLE
Fixed NPE while reading JSON data

### PR DESCRIPTION
**Pull request by thangnc**
- Added checking NPE

### CODE REVIEW

File: src/main/java/org/embulk/output/mailchimp/MailChimpAbstractRecordBuffer.java [hunk 1]
```diff
@@ -267,15 +267,17 @@ public JsonNode apply(JsonNode input)
                 property.put("status", task.getDoubleOptIn() ? PENDING.getType() : SUBSCRIBED.getType());
                 ObjectNode mergeFields = JsonNodeFactory.instance.objectNode();
                 for (String allowColumn : allowColumns.keySet()) {
-                    String value = input.findValue(allowColumns.get(allowColumn)).asText();
+                    JsonNode field = input.findValue(allowColumns.get(allowColumn));
+                    String value = field != null ? field.asText() : "";
```
Comment (sakama) [inline src/main/java/org/embulk/output/mailchimp/MailChimpAbstractRecordBuffer.java]: You can also use
* `input.hasNonNull(allowColumns.get(allowColumn))` 
* `field.isNull()`

**Review by sakama — APPROVED** [review]
Put 1 comment but LGTM

File: src/main/java/org/embulk/output/mailchimp/MailChimpAbstractRecordBuffer.java (hunk 1 above)
Comment (thangnc) [reply src/main/java/org/embulk/output/mailchimp/MailChimpAbstractRecordBuffer.java]: @sakama noted, i will change as your suggestion

**Comment by thangnc** [comment]
@sakama PTAL

**Review by sakama — APPROVED** [review]
LGTM

**Pull request merged**